cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageGdpr.kt

```diff
@@ -97,6 +101,7 @@ internal interface DataStorageGdpr {
         const val GDPR_DATE_CREATED = "sp.gdpr.key.date.created"
         const val GDPR_SAMPLING_VALUE = "sp.gdpr.key.sampling"
         const val GDPR_SAMPLING_RESULT = "sp.gdpr.key.sampling.result"
+        const val GDPR_EXPIRATION_DATE_MESSAGE = "sp.gdpr.key.expiration.date"
```

Comment:
same here, I believe it would be better if we stick to some sort of a convention for the namings of the constants, like `KEY_GDPR_EXPIRATION_DATE`

Reply:
Can expiration date be part of the consent object we store in the local storage rather than a separate key?